packages/core/src/Cardano/types/DelegationsAndRewards.ts

```diff
@@ -16,7 +16,7 @@ export interface Delegatee {
   nextNextEpoch?: StakePool;
 }
 
-export enum StakeKeyStatus {
+export enum StakeCredentialStatus {
```

Comment:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1111/commits/83ef0d1a8f4c9d19d703295658f68b960e77ad53

Reply:
This will end-up with both of those messages (duplicate) in the breaking change list of the changelog
```
refactor!: stakeKeyStatus renamed StakeCredentialStatus

BREAKING CHANGE: StakeKeyStatus type renamed to StakeCredentialStatus
```
It's not terrible, but would be better to do
```refactor!: rename StakeKeyStatus to StakeCredentialStatus

BREAKING CHANGE: rename RewardAccountInfo keyStatus field to credentialStatus
```

Reply:
done!